### PR TITLE
Feature/questionnaire improvements

### PR DESCRIPTION
**Pull request by dgrainge-mwnz**
The IG is now all consistent with Postman and the Questionnaire name/identifier changes yesterday.
It also includes the InitialHealthAssessment and RegularHealthCheck Questionnaires for the first time, clearing up some tech debt from the MVP.

### CODE REVIEW

**Comment by dgrainge-mwnz** [comment]
hi @Kyle good to publish this as the version 0.1.5 of the IG

File: input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+* identifier[0].use = #official
+* identifier[=].value = "COVIDInitialHealthAssessmentQuestionnaire"
+* identifier[=].period.start = "2023-07-19"
+* identifier[+].use = #temp
+* identifier[=].value = "QuestionTemplate-CitC-COVID19-InitialAssessment"
+* identifier[=].period.end = "2023-07-19"
+
+* version = "0.1.5"
+* name = "COVIDInitialHealthAssessmentQuestionnaire"
+* title = "COVID19 Initial Health Assessment Questionnaire"
```
Comment (kyle-mwnz) [inline input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh:16]: This is different to what is currently seeded to the server, which is correct?

File: input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+* status = #draft
+* subjectType = #Patient
+* publisher = "Te Whatu Ora"
+* description = "Questions about a patient's recent COVID-19 history"
```
Comment (kyle-mwnz) [inline input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh:20]: This is different to what is currently seeded to the server, which is correct?

File: input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+* identifier[0].use = #official
+* identifier[=].value = "COVIDRegularHealthCheckQuestionnaire"
+* identifier[=].period.start = "2023-07-19"
+* identifier[+].use = #temp
+* identifier[=].value = "QuestionTemplate-CitC-COVID19-RegularHealthCheck"
+* identifier[=].period.end = "2023-07-19"
+
+* version = "0.1.5"
+* name = "COVIDRegularHealthCheckQuestionnaire"
+* title = "COVID19 Initial Health Assessment Questionnaire"
```
Comment (kyle-mwnz) [inline input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh:16]: Not currently matching

File: input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+* status = #draft
+* subjectType = #Patient
+* publisher = "Te Whatu Ora"
+* description = "Questions about a patient's recent COVID-19 history"
```
Comment (kyle-mwnz) [inline input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh:20]: Not currently matching

**Review by kyle-mwnz — COMMENTED** [review]
Hey @dgrainge-mwnz 

The identifiers and name looks good. I noticed a couple of minor discrepancies with the other fields, which one is correct, server or this PR? It would be good to sort them out, as I am keen to change resources seeding to use the IG to remove the need for storing them in multiple places

File: input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh (hunk 1 above)
Comment (dgrainge-mwnz) [reply input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh:16]: The IG version, name and title shall prevail.  Also the seed resource should be renamed COVIDInitial**Health**AssessmentQuestionnaire.json if we are going to keep using it.

File: input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh (hunk 2 above)
Comment (dgrainge-mwnz) [reply input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh:20]: yep - good catch - IG description is wrong and should be what seed has - I will push a fsh fix

File: input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh (hunk 3 above)
Comment (dgrainge-mwnz) [reply input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh:16]: Line 16 is wrong above but is correct in my view of the branch
* title = "Question Template for COVID-19 Regular Health Check"

However, line 27 in the fsh is incorrect
* useContext.valueCodeableConcept.text = "COVID Initial Health Assessment"

I will push a fix

File: input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh (hunk 4 above)
Comment (dgrainge-mwnz) [reply input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh:20]: hmm on my view of the branch, it shows correctly as 
* description = "Gather the status of a patient who is self isolating"

How can the PR look different to the branch I created it on?

Comment (dgrainge-mwnz) [reply input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh:20]: The seed resource description should match the IG one "Gather the status of a patient who is self isolating"

File: input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh (hunk 2 above)
Comment (dgrainge-mwnz) [reply input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh]: For fsh line 18 above the Publisher produces
```
"subjectType" : [
    "Patient"
  ],
```
So the seed rep is right here even though it doesn't look like valid JSON?

**Review by kyle-mwnz — APPROVED** [review]

**Pull request merged**